packages/react-core/src/helpers/FocusTrap/__tests__/Generated/FocusTrap.test.tsx

```diff
@@ -16,3 +16,21 @@ it('FocusTrap should match snapshot (auto-generated)', () => {
   );
   expect(asFragment()).toMatchSnapshot();
 });
+
+/**
+ * This test was not generated
+ */
+test('Focus trap can have an id added', () => {
```

Comment:
One last thing, if we can let's move this to a separate test file outside the Generated directory.

If you'd want to also add a snapshot test to that new file, we could probably delete the Generated test directory altogether